docs/source/tutorial.md

```diff
@@ -14,7 +12,7 @@ $ docker-compose -f test/fixtures/docker-compose-2orgs-4peers-tls.yaml up
 $ pip3 install virtualenv; make venv
 $ source venv/bin/activate
 $ make install
-$ python docs/sample.py
+$ tox -e py3 -- test/integration/e2e_test.py # Run specified test case
```

Comment:
Test example cannot fully replace sample code.
We should consider to add new sample code.